jwst/outlier_detection/outlier_detection.py

```diff
@@ -98,6 +98,9 @@ def _convert_inputs(self):
         if isinstance(self.inputs, datamodels.ModelContainer):
             self.input_models = self.inputs
             self.converted = False
+        elif isinstance(self.inputs, datamodels.MultiExposureModel):
```

Comment:
Not really relevant at this point in the code, but in general how does `MultiExposureModel` differ from a generic `ModelContainer`? Couldn't the data from multiple exposures, which presumably are all in individual data models of some type, just as easily be stored and processed from within a `ModelContainer` that just contains the list of individual exposure models?